<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendSmsSmppWidget</class>
 <widget class="QWidget" name="SendSmsSmppWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>163</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Destination phone (number):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbDestinationPhone">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblEmpty">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSetupSmpp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string notr="true">Setup SMPP accounts</string>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>14</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Destination TON</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbDestinationTon">
         <item>
          <property name="text">
           <string>Unknown</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>International</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>National</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Network Specific</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Subscriber Number</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Alphanumeric</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Abbreviated</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Destination NPI</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbDestinationNpi">
         <item>
          <property name="text">
           <string>Unknown</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ISDN (E163/E164)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Data (X.121)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Telex (F.69)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Land Mobile (E.212)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>National</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Private</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ERMES</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Internet (IP)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Message template</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbMessageTemplate"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="btnMessageTemplateSelect">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../qHostMonitor.qrc">
           <normaloff>:/img/foldersTree.png</normaloff>:/img/foldersTree.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
